[23653] Documentation debt: TCP sockets and CA extensions configuration (#1115)
eProsima/Fast-DDS-docs: docs/fastdds/use_cases/large_data/large_data.rst

    $> sudo sysctl -w net.core.rmem_max=12582912

Linux also defines per-socket TCP buffer sizes as triplets:

net.ipv4.tcp_wmem = <min> <default> <max> (TCP send buffer)
net.ipv4.tcp_rmem = <min> <default> <max> (TCP receive buffer)

> The middle value is the default used for most connections. If only the global maxima are raised, sockets may still 
> use a small default and drop data during bursts.

Reviewer: ```suggestion
The middle value is the default used for most connections. If only the global maxima are raised, sockets may still 
use a small default and saturate during bursts.
```

Reviewer: If sockets are blocking, as it is our case, the send operation will block instead of drop the data.

Set the current values for sending sockets with:

.. code-block:: bash

>     $> sudo sysctl -w net.ipv4.tcp_rmem="4096 12582912 12582912"

Reviewer: ```suggestion
    $> sudo sysctl -w net.ipv4.tcp_wmem="4096 12582912 12582912"
```

For receiving sockets, the command is:

.. code-block:: bash

>     $> sudo sysctl -w net.ipv4.tcp_wmem="4096 12582912 12582912"

Reviewer: ```suggestion
    $> sudo sysctl -w net.ipv4.tcp_rmem="4096 12582912 12582912"
```

> Increase both the default and max value to match your workload.

Reviewer: ```suggestion
```
